content/java/concepts/math-methods/terms/floor/floor.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Floor'
+Description: 'Returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.'
+Subjects: 
+    -'Code Foundations'
+    -'Computer Science'
+Tags: 
+    - 'Documentation'
+CatalogContent: 
+  - 'learn-java/static-methods-of-the-math-class'
+---
+
+The static function **`.floor()`** included in the `Math` class returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.
```

Comment:
```suggestion
The **`.floor()`** method returns the largest integer value that is less than or equal to the argument.
```

I think the information in the second sentence fits better in the "Syntax" section.

Reply:
This is moved to the Syntax section now.